trolley: 
  # default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  FR_axel_controller: 
    type: effort_controllers/JointPositionController 
    joint: FR_axel_joint
    pid : {p: 100.0, i: 0.001, d: 2.0}

  FL_axel_controller: 
    type: effort_controllers/JointPositionController 
    joint: FL_axel_joint
    pid : {p: 100.0, i: 0.001, d: 2.0}

  RR_wheel_controller: 
    type: velocity_controllers/JointVelocityController 
    joint: RR_wheel_joint
    pid : {p: 10.0, i: 0.001, d: 0.001}

  RL_wheel_controller: 
    type: velocity_controllers/JointVelocityController 
    joint: RL_wheel_joint
    pid : {p: 10.0, i: 0.001, d: 0.001}

  gazebo_ros_control:   
    pid_gains:
      FR_axel_joint : {p: 100.0, i: 0.001, d: 2.0}
      FL_axel_joint : {p: 100.0, i: 0.001, d: 2.0}
      RR_wheel_joint : {p: 10.0, i: 0.001, d: 0.001}
      RL_wheel_joint : {p: 10.0, i: 0.001, d: 0.001}


 